src/reanimated2/layoutReanimation/animationBuilder/commonTypes.ts

```diff
@@ -43,6 +43,7 @@ export type EntryExitAnimationFunction = (
 export type AnimationConfigFunction<T> = (targetValues: T) => LayoutAnimation;
 
 export interface LayoutAnimationsValues {
+  [key: string]: any;
```

Comment:
Why?

Reply:
I changed it to `[key: string]: number;`, but we need to `[key: string]` because I iterate over this object in `/src/reanimated2/layoutReanimation/sharedTransitions/index.ts`

Reply:
Would `[key: string]: unknown` work here instead? Provided we don't actually care about the types of values in this object.